File: sol/2598/2598.py
class Solution:
    def findSmallestInteger(self, nums: List[int], value: int) -> int:
        count = [0] * value
        for x in nums:
            count[x % value] += 1
        v = min(count)
        r = count.index(v)
        return v * value + r

File: sol/2598/2598.cpp
struct Solution {
    int findSmallestInteger(vector<int>& nums, int value) {
        vector<int> count(value, 0);
        for (int x : nums){
            ++count[(x % value + value) % value];
        }
        int r = value, v = nums.size() + 1;
        for (int i = 0; i < value; ++i){
            if (count[i] < v){
                v = count[i];
                r = i;
            }
        }
        return v * value + r;
    }
};
